google/cloud/spanner/bytes.cc

```diff
@@ -55,6 +56,31 @@ static_assert(UCHAR_MAX == 255, "required by base64 encoder");
 
 }  // namespace
 
+// Prints the bytes in the form B"...", where printable bytes are output
+// normally, double quotes are backslash escaped, and non-printable characters
+// are printed as a 3-digit octal escape sequence.
+std::ostream& operator<<(std::ostream& os, Bytes const& bytes) {
+  os << R"(B")";
+  for (auto const byte : Bytes::Decoder(bytes.base64_rep_)) {
+    if (byte == '"') {
+      os << R"(\")";
+    } else if (std::isprint(byte)) {
```

Comment:
`#include <cctype>`

Reply:
Fixed. I don't know how you're so good at spotting missing includes.

Reply:
Who knows, but it probably means the one in values.cc is no longer needed.